app/views/pages/get-started/build-your-first-app/removing-data-from-the-database.liquid

```diff
@@ -0,0 +1,140 @@
+
+The `record_delete` function removes the record, but we need to tell it in which table it is located and what is its ID. We do that in lines 5 and 6. The `table` is hardcoded in this case, but the `id` is referencing the variable we’ve created in the 2nd line.
+
+Finally, we would like some kind of confirmation that the process succeded. In the line 8 we declare that we want the query to return the `id` of the record that has been removed to be sure it worked.
+
+
+## Testing the query using GraphiQL
+
+Before actually using the query on the page, we would probably like to test if it was built properly.
+
+If you are following this tutorial from the start, you already know what to do.
+
+We need to run the following command to start the administration interface for our platformOS instance:
+
+```bash
+pos-cli gui serve <instance alias>
+```
+
+Then, we will be able to go to [http://localhost:3333](http://localhost:3333) in the browser and follow the _Go to GraphiQL_ link.
+
+In the GraphiQL tool we just need to paste our mutation in the left text area, and then expand the bottom section called Query variables where we can pass the ID of the item we want to remove. Assuming we have a database record with the `id` equal to `1`:
+
+```javascript
+{
+  "id": 1
+}
+```
+
+Finally, pressing the big _Play_ button at the top will run the mutation and we should have a confirmation in the right box that the record with the `id` that was `1` was removed.
+
+<img loading="lazy" src="{{ 'images/get-started/todo/gui_graphiql_delete.jpg' | asset_url }}" alt="GraphiQL running the delete mutation for our To do app">
+
+Please remember that all of the operations done in GraphiQL are working on the real data from your database. So if we’d removed a record, it will be gone forever and we can confim this by [using the Records section](/get-started/build-your-first-app/saving-data-to-the-database#browse-the-database-tables-using-platformos-tool) from the `pos-cli` tools.
+
+
+## Using the application interface to delete a record
+
+In this tutorial we’ve prepared a working list of `To do` items. Each of those items have a button next to it, that suppose to mark that item as done, effectively removing it from the list.
+
+The simplest way of making this work would be to use the mutation that we’ve just created. If we remove the record from the database, it will be gone from our list.
+
+To trigger the mutation, we would require a new page that will become a bridge between the user interface and the GraphQL mutation. Let’s create that new page under `/app/views/pages/item/delete.liquid`:
+
+#### /app/views/pages/item/delete.liquid
+
+{% raw %}
+```liquid
+---
+method: post
+---
+
+{% liquid
+	graphql result = 'item/delete', id: context.params.id
+
+	if result.record.id
```

Comment:
@z-x I think if something is wrong with query execution there will be an liquid error.

Reply:
@piotrze - I think if the syntax of the GraphQL is wrong then the `pos-cli` throws an error, but if there is something missing, like an required variable not passed, then the `else` will be helpful.

If something else with the query execution will be wrong then it's probably as you say.